www/app/HardwareController.js

```diff
@@ -1,72 +1,68 @@
 define(['app'], function (app) {
-    app.controller('HardwareController', [ '$scope', '$rootScope', '$location', '$http', '$interval', function($scope,$rootScope,$location,$http,$interval) {
+    app.controller('HardwareController', ['$scope', '$rootScope', '$location', '$http', '$interval', function ($scope, $rootScope, $location, $http, $interval) {
 
-        $scope.SerialPortStr=[];
+        $scope.SerialPortStr = [];
```

Comment:
Please undo all formatting changes.
Use notepad2-mod to edit if your editor keeps doing this